tcms/telemetry/api.py

```diff
@@ -41,5 +41,21 @@ def breakdown(query=None):
 
 
 def _get_field_count_map(test_cases, expression, field):
-    query_set = test_cases.values(field).annotate(count=Count(expression))
-    return [[entry[field], entry['count']] for entry in query_set]
+    query_set_confirmed = test_cases.filter(
+        case_status__name='CONFIRMED'
+    ).values(field).annotate(
+        count=Count(expression)
+    )
+    query_set_not_confirmed = test_cases.exclude(
+        case_status__name='CONFIRMED'
+    ).values(field).annotate(
+        count=Count(expression)
+    )
+    return {
+        'confirmed': _map_query_set(query_set_confirmed, field),
+        'not_confirmed': _map_query_set(query_set_not_confirmed, field)
```

Comment:
The keys in this dictionary are shown as labels in the charts. 'confirmed' -> 'CONFIRMED' (use the value from the DB which you will fetch above because it will be translated). We don't have a string for unconfirmed status so either 'Other' (marked for translation) or something like `! CONFIRMED` / `~ CONFIRMED`.

Reply:
I chose `OTHER`, because `! CONFIRMED` would look weird for a non-programmer, and 
`~ CONFIRMED` looks weird to me